packages/cli/lang/en.lyaml

```diff
@@ -1065,10 +1066,12 @@ en:
       prompts:
         projectDevTargetAccountPrompt:
           createNewSandboxOption: "<Test on a new development sandbox>"
+          createNewDeveloperTestAccountOption: "<Test on a new developer test account>"
           chooseDefaultAccountOption: "<{{#bold}}\U00002757{{/bold}} Test on this production account {{#bold}}\U00002757{{/bold}}>"
-          promptMessage: "[--account] Choose a sandbox under {{ accountIdentifier }} to test with:"
+          promptMessage: "[--account] Choose a {{ accountType }} under {{ accountIdentifier }} to test with:"
           sandboxLimit: "Your account reached the limit of {{ limit }} development sandboxes"
           sandboxLimitWithSuggestion: "Your account reached the limit of {{ limit }} development sandboxes. Run {{ authCommand }} to add an existing one to the config."
+          developerTestAccountLimit: "Your account reached the limit of {{ limit }} developer test accounts."
```

Comment:
@markhazlewood let me know if you'd prefer adding the extra blurb `Run hs auth to add an existing one to the config`